test/test_transforms.py

```diff
@@ -1384,8 +1382,8 @@ def _test_transformation(a, t, s, sh):
             # Compute number of different pixels:
             np_result = np.array(result)
             n_diff_pixels = np.sum(np_result != true_result) / 3
-            # Accept 3 wrong pixels
-            self.assertLess(n_diff_pixels, 3,
+            # Accept 7 wrong pixels
+            self.assertLess(n_diff_pixels, 7,
```

Comment:
Was this test flaky before? Or is this needed due to the change in the center of the transform pixel (the `+0.5` is added in a different location now)